mex-wholebodymodel/library/src/mexmain.cpp

```diff
@@ -39,68 +35,56 @@ using namespace mexWBIComponent;
  * To properly destroy, call ComponentManager::deleteInstance()
  * and then set this pointer to 0.
  */
-static ComponentManager *componentManagerLocalPointerCopy = 0;
```

Comment:
More and less, C++11 can still cause problems in Ubuntu 12.04/14.04 as its ABI is not compatible with the default one of the C++ compiler. For now I think it is easier to just stick to C++98.